config/config.exs

```diff
@@ -7,10 +7,35 @@ config :dora,
 # Configures the endpoint
 config :dora, DoraWeb.Endpoint,
   url: [host: "localhost"],
-  render_errors: [view: DoraWeb.ErrorView, accepts: ~w(json), layout: false],
+  render_errors: [
+    formats: [html: DoraWeb.ErrorHTML, json: DoraWeb.ErrorJSON],
+    layout: false
+  ],
   pubsub_server: Dora.PubSub,
   live_view: [signing_salt: "hbE8k63Z"]
 
+# Configure esbuild (the version is required)
+config :esbuild,
+  version: "0.14.41",
+  default: [
+    args:
+      ~w(js/app.js --bundle --target=es2017 --outdir=../priv/static/assets --external:/fonts/* --external:/images/*),
+    cd: Path.expand("../assets", __DIR__),
+    env: %{"NODE_PATH" => Path.expand("../deps", __DIR__)}
+  ]
+
+# Configure tailwind (the version is required)
+config :tailwind,
+  version: "3.2.4",
+  default: [
```

Comment:
Does it matter that the `esbuild` and  `tailwind` versions don't match the versions set in `mix.exs`?
In here this says `esbuild` uses version `0.14.1` but the `mix.exs` is set to `~ 0.5`. Same thing for `tailwind`

Reply:
Hum, I think they are different things... The version under `mix.exs` I think only refers to the hex package, while the version in the config is actually the "real" version. Also, this part was automatically generated via the `mix phx.new`, so I assume it's expected